scripts/datacube.py

```diff
@@ -394,22 +403,63 @@ def process(
     return ds_merge
```

Comment:
Managed to save one MGRS tile, and tried dragging and dropping the NetCDF into QGIS, but it ends up being georeferenced to (0, 0). We can try to add the georeferencing later (using rioxarray) to make it easier to plot in QGIS.

Reply:
I did quite some work on the xarrays so that they are proper rioxarrays. The algorithm now stores tiles that can be opened in Qgis and friends.